notebooks/gpu-footprint.ipynb

```diff
@@ -0,0 +1,972 @@
+{
```

Comment:
<p>Better move this into Pipfile / requirements.txt instead</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>